.travis.yml

```diff
@@ -3,6 +3,10 @@ language: php
 before_script:
   - composer install
 
+# see https://github.com/travis-ci/travis-ci/issues/4942#issuecomment-159132444
+git:
+  depth: 99999
```

Comment:
Why do we need unshallow clones? I don't see anything requiring it?

Reply:
Somewhere in the CI build you are running these checks against the library itself, and this patch already went further than a shallow clone allows.